### DISCUSSION TITLE
我开发了一个基于maa-cli的gui，关于开源协议的问题想问问

### DISCUSSION THREAD

**Discussion by BoredTape** [MaaAssistantArknights/MaaAssistantArknights#10462] [Q&A] [answered]
是这样的，我开发了一个基于操作maa-cli的gui，提供macOS和linux版本，现在准备开源出来。它在启动的时候会去下载编译后的maa-cli，并且会初始化MaaCore（就是maa-cli的初始化操作）。然后有一个问题想问，我看到maa-cli和maa-core的开源协议是**AGPL-3.0**，而我这个开源的时候想用**GPL-2.0（或3.0）**。我去查了一圈，目前收集到的信息是说可以这样做，最终我还是想问问大伙们的意见。

**Comment by wangl-cc** [comment, marked as answer]
如果你只是调用 maa-cli，没用相关代码的话，应该不会被 AGPL 传染，你想用什么许可证都可以。反之，就必须保持相同的许可证。

**Reply by BoredTape** [reply]
最终，我还是选择AGPL-3.0
